src/inspector_socket_server.cc

```diff
@@ -278,7 +303,8 @@ void InspectorSocketServer::SessionTerminated(int session_id) {
   }
   connected_sessions_.erase(session_id);
   if (connected_sessions_.empty()) {
-    if (was_attached && state_ == ServerState::kRunning
+    if (publish_uid_stderr_
```

Comment:
I suggest passing this all the way into `PrintDebuggerReadyMessage`

Reply:
done.